deployment/ccip/add_chain.go

```diff
@@ -15,13 +15,15 @@ import (
 
 	"github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
 
-	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	"github.com/smartcontractkit/chainlink/deployment"
 	"github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/types"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/ccip_home"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/fee_quoter"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/onramp"
 )
 
+// TEST_LOCATION: integration-tests/ccip-tests/deployment/add_chain_test.go
```

Comment:
If add_chain_test.go does not have any dependency to integration-tests, does it make sense to keep it here? applicable for all memory tests which does not depend on integration-tests

Reply:
yeah I think I agree it should just live right beside add_chain.go, its more of a unit test for a specific changeset component

Reply:
Per our conversation, I'm going to Yoink in the chainlink rpc and rest clients and hopefully that should let me move most of devenv back. Then that should let me restore the memory test here.